### ISSUE TITLE
EditorXR Google poly create mode

### ISSUE THREAD

**Issue by haywirephoenix** [Unity-Technologies/EditorVR#455]
Please support the dream of fully integrating a mesh builder such as Google poly or probuilder.

**Comment by haywirephoenix** [comment]
Are you still there?

**Comment by mtschoen-unity** [comment]
Yes! And we hear you! We also want to live the dream, trust me. As you may be aware, Unity acquired probuilder last year, and the team is hard at work integrating the project. At some point, the goal is to bring Probuilder into EXR.
In the meantime, I invite you to try creating a simple tool to push vertices around or improve on our CreatePrimitive tool. You might be surprised how quick and easy it is to get something working.
